src/main/scala/com.snowplowanalytics/weather/providers/darksky/responses.scala

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright (c) 2015-2019 Snowplow Analytics Ltd. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+ */
+package com.snowplowanalytics.weather
+package providers.darksky
+
+import io.circe.Decoder
+import io.circe.generic.JsonCodec
+import io.circe.generic.extras.semiauto.deriveUnwrappedDecoder
+
+import model.WeatherResponse
+
+object responses {
+
+  @JsonCodec(decodeOnly = true)
```

Comment:
I personally really dislike this annotation style. I'd always prefer `io.circe.generic.semiauto.derive*` methods.

Reply:
Opened new issue.